f3/granite.go

```diff
@@ -84,12 +103,12 @@ func (m GMessage) String() string {
 }
 
 // Computes the payload for a GMessage signature.
-func SignaturePayload(instance uint32, round uint32, step Phase, value ECChain) []byte {
+func SignaturePayload(instance uint32, round uint32, step string, value ECChain) []byte {
```

Comment:
Keep this as Phase and use `.String()` internally.

Reply:
We will probably switch to encoding the Phase as int in the future (if I were to guess).